Tasks/PublishSymbols/PublishSymbols.ps1

```diff
@@ -80,7 +80,7 @@ foreach ($pdbFile in $pdbFiles) {
 Write-Host (Get-LocalizedString -Key "Found {0} symbol files to index." -ArgumentList $pdbFiles.Count)
 
 # Index the sources.
-& $PSScriptRoot\Invoke-IndexSources.ps1 -SymbolsFilePaths $pdbFiles -TreatNotIndexedAsWarning $treatNotIndexedAsWarning
+& $PSScriptRoot\Invoke-IndexSources.ps1 -SymbolsFilePaths $pdbFiles -TreatNotIndexedAsWarning:($treatNotIndexedAsWarning -eq 'true')
```

Comment:
This is related to changing the param on the script called by the handler. Now our input is a string. I changed the param on "Invoke-IndexSources" from a bool to a switch hence the ":" separator between the parameter name and the value instead of the normal " " separator for non-switch params.